docs/topics/wasm/wasm-js-interop.md

```diff
@@ -221,6 +221,46 @@ external class User : JsAny {
 }
 ```
 
+### Array interoperability
+
+You can directly transform JavaScript's `JsArray<T>` into Kotlin's native `Array` or `List` types and vice versa.
+
+To automatically convert `JsArray<T>` to `Array<T>` or the other way around, use one of the available [adapter functions](https://github.com/Kotlin/kotlinx-browser/blob/dfbdceed314567983c98f1d66e8c2e10d99c5a55/src/wasmJsMain/kotlin/arrayCopy.kt).
```

Comment:
Nit: remove unnecessary "automatically". For some reason "automatic conversion" reminds me of "implicit type coercion" which Kotlin chooses not to have :)